src/utils/conditionToDatalog.ts

```diff
@@ -76,6 +80,25 @@ const getTitleDatalog = ({
       ];
     }
   }
+  const currentMatch = /^\s*{current}\s*$/i.test(target);
+  if (currentMatch) {
+    // Can't use this, since it's async
+    // window.roamAlphaAPI.ui.mainWindow.getOpenPageOrBlockUid();
+    const uid = getCurrentPageUid();
```

Comment:
So this will be the current page at time of query, not the current page of the query itself.
If that is the desired results, cool.
But running this in the sidebar will get the current page results.

Reply:
Yea it's meant to be the page of the main window (query drawer doesn't even have a current page). We don't yet have the use case for current page of the query itself